install_config/topics/full_rhv_inventory.adoc

```diff
@@ -63,7 +53,7 @@ openshift_master_htpasswd_users={'myuser': '$apr1$zAhyA9Ko$rBxBOwAwwtRuuaw8OtCwH
 container_runtime_docker_storage_setup_device=/dev/vdb
```

Comment:
line 45: Why do we need a gluster registry? Is that a prereq that we need to list, or is this optional and based on your disks?

Reply:
@kalexand-rh so for RHV there is not a default cloud provider specification. So for this full inventory one of the only solutions that can be provided to have persistent storage for the registry is to use gluster. We previously used nfs but that functionality was deprecated a couple releases ago.

Reply:
Got it. I think that needs to be a prereq in the other module. Something like "To use Red Hat Virtualization, you must [configure GlusterFS](https://docs.openshift.com/container-platform/3.11/install_config/persistent_storage/persistent_storage_glusterfs.html) for persistent storage." I'll update the PR.

Is there anything else that users should know up front about using RHV? Additional configuration requirements, limitations, gotchas?

Reply:
I think we should possibly word it "To us Red Hat Virtualization, you must provide external storage for example GlusterFS for persistent storage of registry images and for application storage.

No additional requirements for RHV though.

Reply:
Thanks @cooktheryan! I updated that step and clarified one of the gluster references in the module with the inventory file.